pkg/cmd/release/create/create_test.go

```diff
@@ -1115,7 +1184,6 @@ func Test_createRun(t *testing.T) {
 			err := createRun(&tt.opts)
 			if tt.wantErr != "" {
 				require.EqualError(t, err, tt.wantErr)
-				return
 			} else {
 				require.NoError(t, err)
 			}
```

Comment:
Not sure why this `return` was here, it seems like it might have been a mistake.

I needed to remove it to get my test to initially fail since my test wants a `SilentError` while also wanting text on stderr 🧐

Reply:
I think this `return` would cause tests that `wantErr` to skip checking `stdout` and `stderr`.  Some of the tests define empty `wantStdout` and `wantStderr`, so removing this `return` would still work for most:

### From `trunk`:

https://github.com/cli/cli/blob/ae87caf8d65f8d5a84646384a68641c0a4d25900/pkg/cmd/release/create/create_test.go#L833-L835

https://github.com/cli/cli/blob/ae87caf8d65f8d5a84646384a68641c0a4d25900/pkg/cmd/release/create/create_test.go#L1115-L1124

### From https://github.com/cli/cli/pull/1552:

You can see the test used to check the resulting release information, too:

https://github.com/cli/cli/blob/0f42c7af4d014b8d711241630099fadd1b588a79/pkg/cmd/release/create/create_test.go#L314-L330